etl/socrata_export/utils/cli.py

```diff
@@ -21,4 +21,9 @@ def get_cli_args():
         action="store_true",
         help="Export the crashes",
     )
+    parser.add_argument(
```

Comment:
so nice to reap the benefits of updating this ETL to selectively update datasets so soon after this change was introduced! 🙌 🎉